src/quacc/calculators/espresso/core.py

```diff
@@ -0,0 +1,31 @@
+from quacc.calculators.espresso.utils import construct_namelist
+from quacc.calculators.espresso.io import _write, _read
+
+# Very simple class that can run calculations ASE style
+# I don't think we even need atoms object here.
+class EspressoCalculator:
```

Comment:
The class name is misleading here, this 'Calculator' is made to deal with other Quantum Espresso binaries (ph.x, neb.x, pp.x, ...) when ASE cannot help (at least not without major tweaking). These binaries do not need atoms objects, most of them are always done after a pw.x calculation. The plan here is to let quacc use ase.calculators.espresso without doing major changes to do pw.x jobs. But each time a user wants to integrate other binaries in a workflow quacc will have to use this special Calculator (which name should probably be changed)

I think there will be no real advantage to make this class inherit from genericfileio?

Reply:
Ah, thank you for the explanation! I've never run QE calculations before, so this is very helpful. I will take another look with this mind after Thanksgiving :)

Reply:
QE has 30+ binaries like that. Although it can be quite dense, it is somewhat perfect for workflows... (I am actually surprised that no one did that before, maybe I am not so well informed). Enjoy Thanksgiving!